pdsApp/src/app/controllers/UserCommunicate.java

```diff
@@ -259,6 +268,72 @@ public boolean getConnection(String login, String pwd){
     return userConnected;
   }
 
+  private void getAdressAgency(String login, String pass) {
```

Comment:
This method should be in the "address" Model!
Ce fichier finira à 100 000 lignes si on code toutes les methodes ici lol .
Faudra mettre chaque methode dans son propre model